Categories/Category/Monoidal/Braided/Properties.agda

```diff
@@ -0,0 +1,85 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category using (Category)
+open import Categories.Category.Monoidal using (Monoidal)
+open import Categories.Category.Monoidal.Braided using (Braided)
+
+module Categories.Category.Monoidal.Braided.Properties
+  {o ℓ e} {C : Category o ℓ e} {M : Monoidal C} (BM : Braided M) where
+
+open import Data.Product using (_,_)
+
+open import Categories.Category.Monoidal.Properties M
+open import Categories.Functor using (Functor)
+open import Categories.Morphism.Reasoning C
+open import Categories.NaturalTransformation.NaturalIsomorphism.Properties
+  using (push-eq)
+
+open Category C
+open Braided BM
+open Commutation
+open MonoidalReasoning
+
+private
+  variable
+    X Y Z : Obj
+
+  -- A shorthand for the braiding
+  B : ∀ {X Y} → X ⊗₀ Y ⇒ Y ⊗₀ X
+  B {X} {Y} = braiding.⇒.η (X , Y)
+
+  -- sstucki: the structure of this proof is not obvious, and there
+  -- are no references to the literature.  It would be nice to give a
+  -- high-level, informal explanation of the proof.  For example, what
+  -- is the intuition behind the following lemma?  Why is it
+  -- necessary/useful in the final proof?
```

Comment:
@sstucki 
I wrote an explanation of the proof and drew a diagram.
Could you please give me write access to your branch so that I may push onto this PR?